src/node.ts

```diff
@@ -0,0 +1,59 @@
+import { PaymentSdkConfigType } from "./types";
+
+import { XcooBeePaymentSDK } from "./XcooBeePaymentSDK";
+
+import { logoSrc } from "./config";
+
+/**
+ * Implements node version of Base64Interface to manipulate base64 strings
+ *
+ * @implements Base64Interface
+ */
+export const base64 = {
```

Comment:
This one is needed when user wants to set custom QR generator. They use prebuilt base64 interfaces instead of implementing custom ones

Reply:
@shablenko-sednor Sounds like there would not be a need for a custom base64 implementation. The user just needs the correct one based on whether they are using node or a browser. There may be a way we can auto-select it for them. I know it works with Webpack. I'm not sure if it works with `tsc`.

To do it, you create an embedded package.
1) Add a directory for the package (`src/base64`)
2) Add a `package.json` file to directory.
```
{
  "private": true,
  "browser": "browserBase64",
  "main": "nodeBase64"
}
```
Something along those lines. But it may not work with `tsc`.

Another alternative is to use https://www.npmjs.com/package/abab. I believe it should work for node or a browser. With this solution, the end user will not have to worry about which implementation of Base64 to use. All they need is to implement a custom solution for the QrGenerator.